ui/raidboss/data/06-ew/dungeon/lapis_manalis.txt

```diff
@@ -0,0 +1,283 @@
+45.3 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+51.7 "Left/Right Slam" sync / 1[56]:[^:]*:Albion:(802E|802D):/
+60.3 "Knock on Ice" sync / 1[56]:[^:]*:Albion:7A7E:/
+62.3 "Knock on Ice" sync / 1[56]:[^:]*:Albion:7A7F:/
+68.6 "Icebreaker" sync / 1[56]:[^:]*:Albion:7A81:/
+71.6 "Icy Throes" sync / 1[56]:[^:]*:Albion:7A82:/
+77.8 "Icy Throes" sync / 1[56]:[^:]*:Albion:(800F|7A83):/
+78.2 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+82.8 "Icy Throes" sync / 1[56]:[^:]*:Albion:7FB9:/
+86.8 "Roar of Albion" sync / 1[56]:[^:]*:Albion:7A84:/
+99.9 "Call of the Mountain" sync / 1[56]:[^:]*:Albion:7A7C:/
+105.3 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+110.9 "Wildlife Crossing 1" sync / 00:0839::A rumbling in the distance draws closer.../ window 30,30 duration 15
+112.1 "Left/Right Slam" sync / 1[56]:[^:]*:Albion:(802E|802D):/
+119.6 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+124.9 "Wildlife Crossing 2" sync / 00:0839::More rumbling can be heard in the distance.../ window 30,30 duration 15
+125.9 "Left/Right Slam" sync / 1[56]:[^:]*:Albion:(802E|802D):/
+135.1 "Albion's Embrace" sync / 1[56]:[^:]*:Albion:7A85:/
+143.1 "Knock on Ice" sync / 1[56]:[^:]*:Albion:7A7E:/
+145.1 "Knock on Ice" sync / 1[56]:[^:]*:Albion:7A7F:/
+151.2 "Icebreaker" sync / 1[56]:[^:]*:Albion:7A81:/
+154.2 "Icy Throes" sync / 1[56]:[^:]*:Albion:7A82:/
+160.4 "Icy Throes" sync / 1[56]:[^:]*:Albion:(800F|7A83):/
+160.8 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+165.4 "Icy Throes" sync / 1[56]:[^:]*:Albion:7FB9:/
+169.4 "Roar of Albion" sync / 1[56]:[^:]*:Albion:7A84:/ window 30,30 jump 86.8
+182.6 "Call of the Mountain" sync / 1[56]:[^:]*:Albion:7A7C:/
+188.0 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+193.4 "Wildlife Crossing 1" sync / 00:0839::A rumbling in the distance draws closer.../ window 30,30 duration 15
+194.6 "Left/Right Slam" sync / 1[56]:[^:]*:Albion:(802E|802D):/
+202.2 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+207.2 "Wildlife Crossing 2" sync / 00:0839::More rumbling can be heard in the distance.../ window 30,30 duration 15
+208.3 "Left/Right Slam" sync / 1[56]:[^:]*:Albion:(802E|802D):/
+217.5 "Albion's Embrace" sync / 1[56]:[^:]*:Albion:7A85:/
+225.5 "Knock on Ice" sync / 1[56]:[^:]*:Albion:7A7E:/
+227.5 "Knock on Ice" sync / 1[56]:[^:]*:Albion:7A7F:/
+233.8 "Icebreaker" sync / 1[56]:[^:]*:Albion:7A81:/
+236.8 "Icy Throes" sync / 1[56]:[^:]*:Albion:7A82:/
+243.0 "Icy Throes" sync / 1[56]:[^:]*:Albion:(800F|7A83):/
+243.4 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+248.1 "Icy Throes" sync / 1[56]:[^:]*:Albion:7FB9:/
+252.1 "Roar of Albion" sync / 1[56]:[^:]*:Albion:7A84:/
+265.3 "Call of the Mountain" sync / 1[56]:[^:]*:Albion:7A7C:/
+
+#~~~~~~~~~~~~~~~#
+# Galatea Magna #
+#~~~~~~~~~~~~~~~#
+
+1000.0 "--sync--" sync / 00:0839::The Forum Messorum will be sealed off/ window 1000,1
+1010.2 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F71:/
+1015.2 "Waxing/Waning Cycle" sync / 1[56]:[^:]*:Galatea Magna:(7A91|7F6E):/
```

Comment:
Same thing here re Left/Right Slam, can you call this Waxing Cycle/Waning Cycle and add a translation?